python/cudf/cudf/core/column/string.py

```diff
@@ -3988,6 +3988,21 @@ def _mimic_inplace(self, other_col, inplace=False):
         out = super()._mimic_inplace(other_col, inplace=inplace)
         return out
 
+    def view(self, dtype):
+        dtype = np.dtype(dtype)
+        if (
+            self.base_children[0][self.offset]
+            * self.base_children[1].dtype.itemsize
+        ) % dtype.itemsize:
+            raise TypeError(
+                "Slice data size must divide evenly into desired dtype size"
+            )
+        else:
```

Comment:
Can reduce nesting by removing the `else` statement, since the previous block raises an error.